marlowe-runtime/proxy/Language/Marlowe/Runtime/Proxy.hs

```diff
@@ -10,76 +10,73 @@
 module Language.Marlowe.Runtime.Proxy
```

Comment:
Changes in this module:

- Moved the server source to the output of the component
- Added an `inProc` flavour of the component, which directly communicates with the downstream servers via `ServerSource`es. This is the variant used in `marlowe-runtime`